client/components/wpadmin-auto-login/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External Dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { get } from 'lodash';
+
+/**
+ * Try to preload image pixel via going through wp-admin login.
+ * If SSO login pass-thru is enabled, you will be redirected to pixel
+ */
+function tryLogin( requestUrl, initalDelay, attempt ) {
+	const image = new Image();
+	image.src = requestUrl;
+
+	if ( attempt < 9 ) {
+		image.onerror = function() {
```

Comment:
Hmm, this is me being paranoid, but can we try adding the onerror before setting src?